lib/parallel_tests/cli.rb

```diff
@@ -216,6 +224,7 @@ def parse_options!(argv)
         opts.on("--unknown-runtime [FLOAT]", Float, "Use given number as unknown runtime (otherwise use average time)") { |time| options[:unknown_runtime] = time }
         opts.on("--first-is-1", "Use \"1\" as TEST_ENV_NUMBER to not reuse the default test environment") { options[:first_is_1] = true }
         opts.on("--verbose", "Print more output") { options[:verbose] = true }
+        opts.on("--features_from_process", "Write features to text file") { options[:features_from_process] = true }
```

Comment:
`features-from-process`

Reply:
the name does not really convey a purpose/feature to me and neither does the description ...